website/versioned_docs/version-25.x/GettingStarted.md

```diff
@@ -15,7 +15,11 @@ Or [`npm`](https://www.npmjs.com/package/jest):
 npm install --save-dev jest
 ```
 
-Note: Jest documentation uses `yarn` commands, but `npm` will also work. You can compare `yarn` and `npm` commands in the [yarn docs, here](https://yarnpkg.com/en/docs/migrating-from-npm#toc-cli-commands-comparison).
+:::note
+
+Jest documentation uses `yarn` commands, but `npm` will also work. You can compare `yarn` and `npm` commands in the [yarn docs, here](https://yarnpkg.com/en/docs/migrating-from-npm#toc-cli-commands-comparison).
+
+:::
```

Comment:
I think it would be better to use `npm2yarn` plugin instead. See #12473, in which only the latest version of documentation was touched. It would be great to make similar changes in all versioned docs as well.